_posts/2022-10-18-torch-linalg/torch-linalg.Rmd

```diff
@@ -0,0 +1,632 @@
+---
+title: "Five ways to do least squares (with torch)"
+description: >
+  Get to know torch's linalg module, all while learning about different ways to do least-squares regression from scratch. This post, again, is a condensed version of the corresponding chapter in the forthcoming book, Deep Learning and Scientific Computing with R torch, to be published by CRC Press. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchlinalg
+date: 2022-10-18
+categories:
+  - Torch
+  - R
+  - Concepts
+  - Tabular data
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/squirrel.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+*Note: This post is a condensed version of a chapter from part three of the forthcoming book, Deep Learning and Scientific Computing with R torch. Part three is dedicated to scientific computation beyond deep learning. As throughout the book, but -- if possible -- to an even higher degree, there I focus on the underlying concepts, striving to explain them in as "verbal" a way as I can. This does not mean skipping the equations; it means taking care to explain why they are the way they are.*
```

Comment:
> As throughout the book, but -- if possible -- to an even higher degree, there I focus on the underlying concepts

You've lost me a little bit here with the caveats. To a higher degree than what? I think you might just simplify this to "Throughout the book I focus on underlying concepts…” Are you trying to say that in the book, as opposed to in this post, you focus on underlying concepts? If that's the case, I'd state that in as straightforward a manner as possible. 🙂

Reply:
no, I meant in part 3 even more than parts 1 & 2 ... you're right, I'll just throw it out completely!